django_redis/client/default.py

```diff
@@ -113,14 +124,14 @@ def disconnect(self, index=0, client=None):
 
     def set(
         self,
-        key,
-        value,
-        timeout=DEFAULT_TIMEOUT,
-        version=None,
-        client=None,
-        nx=False,
-        xx=False,
-    ):
+        key: Any,
+        value: Any,
+        timeout: Optional[float] = DEFAULT_TIMEOUT,
+        version: Optional[int] = None,
+        client: Redis = None,
```

Comment:
```suggestion
        client: Redis = Optional[None],
```

Reply:
I think there is a little typo here, but I understood, it is in line with the other suggestions

Reply:
lol, yeah, I should have just marked one and left you to `s/Redis/Optional[Redis]/` :sweat_smile:

Reply:
I see you are a `sed` user. Nice to meet you mate

Reply:
done